### PR TITLE
feat: add global transition disable switch

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: www/src/pages/testing.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { graphql } from 'gatsby';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import Layout from '../components/Layout';
+import Example from '../components/Example';
+
+const propTypes = {
+  location: PropTypes.object.isRequired,
+  data: PropTypes.shape({
```
Comment (taion) [inline www/src/pages/testing.js]: why do we need all this 🤨

Comment (jquense) [reply www/src/pages/testing.js]: we don't i just copy/pasted from the other page

**Comment by piotr-cz** [comment]
IMHO docs should be updated with [info](https://github.com/reactjs/react-transition-group/blob/v4.1.0/www/src/pages/testing.js#L28-L35) how to use this feature

**Pull request merged**